spec/javascripts/support/jasmine_helper.rb

```diff
@@ -1,15 +0,0 @@
-#Use this file to set/override Jasmine configuration options
```

Comment:
@matthewhuff89 @kaylorburdell - Not sure about deleting this, is it gonna interfere with us running Jasmine tests in the future? We still want to test our JavaScript.